concent_api/core/subtask_helpers.py

```diff
@@ -1,10 +1,13 @@
-from base64                     import b64encode
+from base64 import b64encode
 from logging import getLogger
+from typing import Any
 from typing import List
 from typing import Optional
+from typing import Union
 
-from django.db.models           import Q
-from django.utils               import timezone
+import django
```

Comment:
Please use the `from django.db.models import Model` form. It's already used for other names imported from `django` in this module and importing stuff in two different ways in the same module can be confusing so avoid it unless there's no other way.

Reply:
done